apps/core/templates/base.html

```diff
@@ -30,6 +32,24 @@
             <link rel="alternate" hreflang="en" href="{{ page.full_url|hreflang_url:page.locale.language_code }}" >
         {% endif %}
 
+        {% comment %}
+            Twitter summary card - see https://dev.twitter.com/cards/types/summary
+            and https://dev.twitter.com/cards/getting-started
+        {% endcomment %}
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="@wagtailcms" />
+        <meta name="twitter:title" content="{{ page.title }}" />
+        <meta name="twitter:description" content="{% if page.search_description %}{{ page.search_description }}{% endif %}">
+        <meta name="twitter:image" content="{{ current_site.root_url }}{% static 'images/social-image.png' %}">
```

Comment:
I doubt this would work as our Webpack tooling doesn’t have support for images. I’ll move the file to the `public` folder later.

Reply:
My bad, this works, just needs to use `manifest` for correctness. It wasn’t working in Gitpod for other reasons (site record)